### ISSUE TITLE
SVD implementation

### ISSUE THREAD

**Issue by Lundez** [Kotlin/multik#85]
It's great that LU / QR / Hessenberg etc was added, but I think SVD would be golden to have as an additional linalg component. A lot of the algorithms from Python bases themself on the SVD decomposition, especially the compact variant.

Is this on the agenda to be added?

**Comment by devcrocod** [comment]
Yes, it will be added. I can't say yet if it will be in the next release, since we are first adding a multiplatform.

**Comment by Lundez** [comment]
Thanks for the update @devcrocod ! 🤗

**Comment by devcrocod** [comment]
`_gesdd` requires `quadmath`, but so far I haven't built this library for Android (related to issue #127)
